packages/devtools_app/lib/src/framework/initializer.dart

```diff
@@ -53,7 +53,7 @@ class Initializer extends StatefulWidget {
   final bool allowConnectionScreenOnDisconnect;
 
   @override
-  _InitializerState createState() => _InitializerState();
+  State createState() => _InitializerState();
```

Comment:
This should have the widget type: `State<Initializer>`. Here and elsewhere.

This is what the `stful` quick fix in vs code and intelliJ do by default:
```dart
class MyWidget extends StatefulWidget {
  const MyWidget({super.key});

  @override
  State<MyWidget> createState() => _MyWidgetState();
}

class _MyWidgetState extends State<MyWidget> {
  @override
  Widget build(BuildContext context) {
    return const Placeholder();
  }
}
```